src/tests/ecalc_cli/snapshots/test_app/test_csv_default/test.csv

```diff
@@ -1,13 +1,12 @@
-2031-01-01 00:00:00,1,        260635.10000,    1279169392.00000,            32.52178,          2965.53300,           570.79090,       2801381.00000,          7000.00000,69.02248,10.97337,nan,nan,            32.52178,          2965.53300,             0.00000,             0.00000,1,        260635.10000,    1279169392.00000,            32.52178,          2965.53300,           570.79090,       2801381.00000,          7000.00000,69.02248,10.97337,nan,nan,            32.52178,          2965.53300,             0.00000,             0.00000,             1.00000,1,        203635.10000,     774131952.00000,            32.52178,          2965.53300,           445.96090,       1695349.00000,            67.47822,1,         50000.00000,     476911440.00000,           109.50000,       1044436.00000,             0.00000,0,1,             6.89000,         23762.14000,             6.89000,           570.29140,             0.00000,0,1,             4.50746,         19027.96000,             4.50746,           456.67100,         14000.00000,            10.00000,           200.00000,         18811.88000,1,            11.80000,         47412.40000,            11.80000,          1137.89800,1,          7000.00000,      28126000.00000,            15.33000,         61595.94000,1,             9.32432,         33361.36000,             9.32432,           800.67260
+2020-01-01 00:00:00,1,        310830.00000,     113763780.00000,            27.71452,           243.44430,           680.71770,        249142.70000,         12500.00000,            54.45742,             8.65778,54.45742,8.657777,            27.71452,           243.44430,             0.00000,             0.00000,1,        310830.00000,     113763780.00000,            27.71452,           243.44430,           680.71770,        249142.70000,         12500.00000,            54.45742,             8.65778,54.45742,8.657777,            27.71452,           243.44430,             0.00000,             0.00000,             1.00000,1,        173830.00000,      63621780.00000,            27.71452,           243.44430,           380.68770,        139331.70000,            72.28548,1,        130000.00000,      47580000.00000,           284.70000,        104200.20000,             0.00000,0,1,             4.89000,          1789.74000,             4.89000,            42.95376,             0.00000,0,1,             5.02452,          1838.97300,             5.02452,            44.13536,         17000.00000,            10.00000,           200.00000,         18811.88000,1,            11.80000,          4318.80000,            11.80000,           103.65120,1,          7000.00000,       2562000.00000,            15.33000,          5610.78000,1,             6.00000,          2196.00000,             6.00000,            52.70400
```

Comment:
cumulative are now the cumulative volume at the end of the period (or timestep here).